packages/ui-rating/src/Rating/index.tsx

```diff
@@ -77,38 +78,27 @@ class Rating extends Component<RatingProps> {
     const { valueNow, iconCount, valueMax } = this.props
 
     // prevent divide by zero errors
-    // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-    const max = valueMax > 0 ? valueMax : iconCount
+    const max = !!valueMax && valueMax > 0 ? valueMax : iconCount
 
-    // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-    const filledIcons = Math.round((valueNow * iconCount) / max)
+    const filledIcons = Math.round((valueNow! * iconCount!) / max!)
 
     // Handle edge case where valueNow is greater than valueMax
-    // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-    if (filledIcons > iconCount) {
+    if (filledIcons > iconCount!) {
       return iconCount
```

Comment:
Add a `!` here too because it will never return undefined (iconCount has a default). This will make the whole function type fixed and will say it always returns a number. Because of this you can remove the `!` from `this.filled!` in other places where it is used.

```suggestion
      return iconCount!
```